### PR TITLE
Allow installation of nelmio/cors-bundle >= v1.3.3

### PR DESCRIPTION
**Pull request by guillaumelecerf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "kriswallsmith/buzz": ">=0.9",
         "tedivm/stash-bundle": "0.4.*",
         "sensio/distribution-bundle": "~3.0",
-        "nelmio/cors-bundle": "1.3.*",
+        "nelmio/cors-bundle": "1.4.*",
```
Comment (andrerom) [inline composer.json]: Maybe rather change to `^1.3.3`, also allowing 1.3.3 as long as we don't use any features of 1.4 yet.

**Comment by guillaumelecerf** [comment]
BTW why is `composer.lock` versionned ?

**Comment by andrerom** [comment]
Because all travis jobs where failing as of a few weeks ago because of most likely IP6 issues with packagist.org, and composer.lock is one way to reduce traffic against it. Containers + composer cache would be another, however mysql is insanely slow on travis containers so not an option.

**Comment by andrerom** [comment]
+1

**Comment by andrerom** [comment]
Thanks @guillaumelecerf, merged now :)

**Comment by guillaumelecerf** [comment]
Thanks ;)

**Pull request merged**